core/shared/platform/common/posix/posix_socket.c

```diff
@@ -260,7 +260,7 @@ os_socket_recv_from(bh_socket_t socket, void *buf, unsigned int len, int flags,
                     bh_sockaddr_t *src_addr)
 {
     struct sockaddr_storage sock_addr = { 0 };
-    socklen_t socklen = sizeof(sock_addr);
+    socklen_t socklen = sizeof(struct sockaddr_storage);
```

Comment:
Is there issue found in the original code? Had better not change the original implementation?

Reply:
Restored the original implementation.